.github/workflows/build_dependabot_bundler_pr.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow is based on https://github.com/dependabot/dependabot-actions-workflow
+
+name: Build Dependabot Bundler PR
+on:
+  push:
+    branches:
+      - "dependabot/bundler**"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Golang
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19
+
+      - name: run mod check
```

Comment:
Are lines 15-21 the only lines added from the original PR? Or are there other changes?

Reply:
Also fixed the `- name: Commit and copy changes` item